### DISCUSSION TITLE
Click outside

### DISCUSSION THREAD

**Discussion by mawsyh** [klinecharts/KLineChart#475] [Ideas]
Can we add a new API to trigger when user clicks outside of an overlay ?

**Comment by liihuu** [comment]
`onDeselected`?

**Reply by mawsyh** [reply]
Deselect is not what I have in my mind, Imagine we have a custom overlay with multiple parts something like the picture below,
![image](https://github.com/liihuu/KLineChart/assets/90469267/101212b3-d977-48e5-b623-b59086ca9e6c)
in order to capture mouse click outside the used figures "Deselect" simply won't work (is not coded to do that)
I suggest we add something inside the "CheckEvenOn" functions, I have to take a deeper look tho.
